fredapi/fred.py

```diff
@@ -28,7 +31,8 @@ class Fred:
 
     def __init__(self,
                  api_key=None,
-                 api_key_file=None):
+                 api_key_file=None,
+                 proxies=None):
```

Comment:
could you please add an explanation in the docstring below about how `proxies` can be used?

Reply:
I can take care, sure. I will also add a parameter to specify ssl verification. I'll push in the next couple days.

Reply:
Done here: https://github.com/mortada/fredapi/pull/71